From 505a6a8befcd0c646301100c465758795c1317d2 Mon Sep 17 00:00:00 2001
From: Jesus Alberto Paz Jimenez <alberto.pazjimenez@gmail.com>
Date: Mon, 28 Apr 2014 17:03:32 +0200
Subject: [PATCH] Update zenroom2.vim
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

VIM - Vi IMproved 7.4 under cygwin generates the following errors

>Se ha detectado un error al procesar /home/alberto.paz/.vim_runtime/sources_non_forked/vim-zenroom2/plugin/zenroom2.vim:
>línea   65
>E128: Function name must start with a capital or "s:": g:zenroom_goyo_before()
>línea   67
>E133: ":return" no está dentro de una función
>línea   74
>E193: ¡":endfunction" no está dentro de una función!
>línea   76
>E128: Function name must start with a capital or "s:": g:zenroom_goyo_after()
>línea   78
>E133: ":return" no está dentro de una función
>línea   88
>E193: ¡":endfunction" no está dentro de una función!
>línea   90
>E700: Función desconocida: g:zenroom_goyo_before
>E700: Función desconocida: g:zenroom_goyo_after
---
 sources_non_forked/vim-zenroom2/plugin/zenroom2.vim | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sources_non_forked/vim-zenroom2/plugin/zenroom2.vim b/sources_non_forked/vim-zenroom2/plugin/zenroom2.vim
index a9781c7..2ea60f0 100644
--- a/sources_non_forked/vim-zenroom2/plugin/zenroom2.vim
+++ b/sources_non_forked/vim-zenroom2/plugin/zenroom2.vim
@@ -62,7 +62,7 @@ function! s:markdown_room()
     exec( "hi StatusLineNC " . l:highlightfgbgcolor )
 endfunction
 
-function! g:zenroom_goyo_before()
+function! g:Zenroom_goyo_before()
     if !has("gui_running")
         return
     endif
@@ -73,7 +73,7 @@ function! g:zenroom_goyo_before()
     endif
 endfunction
 
-function! g:zenroom_goyo_after()
+function! g:Zenroom_goyo_after()
     if !has("gui_running")
         return
     endif
@@ -87,4 +87,4 @@ function! g:zenroom_goyo_after()
     endif
 endfunction
 
-let g:goyo_callbacks = [ function('g:zenroom_goyo_before'), function('g:zenroom_goyo_after') ]
+let g:goyo_callbacks = [ function('g:Zenroom_goyo_before'), function('g:Zenroom_goyo_after') ]
-- 
2.0.3

